sdk/monitor/azquery/README.md

```diff
@@ -1,35 +1,39 @@
 # Azure Monitor Query client module for Go
 
-The Azure Monitor Query client library is used to execute read-only queries against [Azure Monitor][azure_monitor_overview]'s two data platforms:
+The Azure Monitor Query client module is used to execute read-only queries against [Azure Monitor][azure_monitor_overview]'s two data platforms:
 
 - [Logs][logs_overview] - Collects and organizes log and performance data from monitored resources. Data from different sources such as platform logs from Azure services, log and performance data from virtual machines agents, and usage and performance data from apps can be consolidated into a single [Azure Log Analytics workspace][log_analytics_workspace]. The various data types can be analyzed together using the [Kusto Query Language][kusto_query_language]. See the [Kusto to SQL cheat sheet][kusto_to_sql] for more information.
 - [Metrics][metrics_overview] - Collects numeric data from monitored resources into a time series database. Metrics are numerical values that are collected at regular intervals and describe some aspect of a system at a particular time. Metrics are lightweight and capable of supporting near real-time scenarios, making them particularly useful for alerting and fast detection of issues.
 
-[Source code][azquery_repo] | [Package (pkg.go.dev)][azquery_pkg_go] | [Product documentation][monitor_docs] | [Samples][azquery_pkg_go_samples]
+Key links:
+
+[Source code][azquery_repo] | [API reference documentation][azquery_pkg_go_docs] | [REST API documentation][monitor_rest_docs] | [Product documentation][monitor_docs] | [Samples][go_samples]
```

Comment:
Not sure if I like the changes to the links. "Key links" heading is not found in other sdks. 

Samples should link to the samples on the pkg.go website, not the general sample website. Should still keep the Package pkg.go.dev link.

Reply:
Thanks for feedback. Still converging on a common style here and we may want to consider whether the main context for the README is the pkg.go.dev site or the GitHub repo. If the former, the Package link is circular.
I removed "Key links:" heading, and restored the Package pkg.go.dev link and original "Samples" link.